provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -792,6 +792,7 @@ tasks:
     vars:
       SCENARIO: '{{.SCENARIO | default "keycloak.scenario.authentication.AuthorizationCode"}}'
       USERS: "{{.USERS | default 20000}}"
+      CLIENTS: "{{.CLIENTS | default 10000}}"
```

Comment:
Is this used somewhere? Should we remove from the Taskfile if not?

Reply:
I use the task file to trigger the runs when I need it. I want to keep it in sync with the nightly defaults.